# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1263244035 18000
#      Mon Jan 11 16:07:15 2010 -0500
# Node ID 81e88250bf422472e868c965060d07be5eeecd81
# Parent  614ad9e7a17bd0135d2ebe4f4d4e70203dcc35ef
urlwrite: avoid using errno as a local variable

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,13 @@
+2010-01-11  John W. Eaton  <jwe@octave.org>
+
+	* DLD-FUNCTIONS/urlwrite.cc (curl_handle::lasterror):
+	Avoid using errno as a local variable name.
+
 2010-01-11  Jaroslav Hajek  <highegg@gmail.com>
 
 	* DLD-FUNCTIONS/lu.cc (Fluupdate): Add tests for LU updating.
 
 2010-01-11  Tatsuro MATSUOKA  <tmacchant@yahoo.co.jp>
 
 	* Makefile.am (OCTINTERP_LINK_DEPS): Include FT2_LDFLAGS and
 	FT2_LIBS in the list.
diff --git a/src/DLD-FUNCTIONS/urlwrite.cc b/src/DLD-FUNCTIONS/urlwrite.cc
--- a/src/DLD-FUNCTIONS/urlwrite.cc
+++ b/src/DLD-FUNCTIONS/urlwrite.cc
@@ -246,21 +246,21 @@ public:
 
   bool is_valid (void) const
     {
       return rep->is_valid ();
     }
 
   std::string lasterror (void) const
     {
-      CURLcode errno;
+      CURLcode errnum;
 
-      curl_easy_getinfo (rep->handle(), CURLINFO_OS_ERRNO, &errno);
+      curl_easy_getinfo (rep->handle(), CURLINFO_OS_ERRNO, &errnum);
       
-      return std::string (curl_easy_strerror (errno));
+      return std::string (curl_easy_strerror (errnum));
     }
 
   void set_ostream (std::ostream& os) const
     {
       setopt (CURLOPT_WRITEDATA, static_cast<void*> (&os));
     }
 
   void set_istream (std::istream& is) const
